src/main/java/seedu/address/model/group/Group.java

```diff
@@ -19,6 +21,7 @@
 
     // Data fields
     private final Set<Person> persons = new HashSet<>();
+    private final UniqueTaskList tasks = new UniqueTaskList();
```

Comment:
to check later

Reply:
This one also no need to change for now 👍 . I was thinking if we should use `ObservableList` instead. But we can keep it like this for now.